packages/components/src/radio-control/style.scss

```diff
@@ -5,6 +5,10 @@
 	.components-base-control__help {
 		margin-top: 0;
 	}
+
+	.components-base-control__field {
+		margin-bottom: 4px;
```

Comment:
This seems inconsistent with other controls (text, checkbox, toggle). Shouldn't we keep the same value for all types of controls

Reply:
The margin-bottom applied to the `.components-base-control__field` is varied quite a bit already. 

Some examples:
1. `.components-font-size-picker__select` (and a number of others) sets mb to `0`
2. `.components-toggle-control` sets its margin-bottom to `12px`
3. `.components-base-control` sets its margin-bottom to `8px`

Reply:
Any reasons for these. I think we should align them all, otherwise we'd have weird results depending on the composition of the different components.

Reply:
I'll do an audit and see where we can improve. I believe it's due to elements within the `.components-base-control__field` that have varying spacing.

Reply:
> I believe it's due to elements within the `.components-base-control__field` that have varying spacing

@youknowriad + @richtabor 

👋 Hallo! It sounds like the component's varying implementation is more the root of the problem (as you folks have touched upon). This particular PR/fix was just for a symptom.

Looking at the `RadioControl` component, it only ever renders a single `BaseControl`, so I think nullifying the the margin is fine in this context. After doing a quick audit, it looks like several other custom styles of `.components-base-control__field` nullify the margin to `0` as well. (Not all... but several)

Let us know what you discover in your audit 😊